test/e2e/reconfigure_test.go

```diff
@@ -46,10 +46,6 @@ func (tc *testContext) reconfigurationTest(t *testing.T) {
 	_, err = tc.client.K8s.CoreV1().Nodes().Patch(context.TODO(), machineNodes[0].Name, types.JSONPatchType,
 		patchData, metav1.PatchOptions{})
 	require.NoError(t, err)
-	// Ensure operator communicates to OLM that upgrade is not safe when processing Machine nodes
-	err = tc.validateUpgradeableCondition(metav1.ConditionFalse)
```

Comment:
> If WICD fixes the change by reconciling the node, before WMCO is able to reconcile the Machine or windows instance ConfigMap, WMCO's controllers will return early, and not change the condition state.

I don't follow the commit message. Isn't this a bug then? I guess I am not understanding the timing issue here.

Reply:
Moved to separate PR